javascript/linked-list/stacks-and-queues/stack-queue-psuedo.test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// const { Node, Stack } = require('./stacksandqueues');
+const { PsuedoQueue } = require('./stack-queue-pseudo');
+
+describe('Stack-Queue-Psuedo', () => {
+  test('Can succesfully enqueue a node', () => {
+    const testPQueue = new PsuedoQueue(null);
+    testPQueue.enqueue('dummynode');
+    expect(testPQueue.top.value).toEqual('dummynode');
```

Comment:
Look at the structure of your PsuedoQueue class again; it does not have a top, but it does have s1 and s2 stacks that each have a top.